wordpress-mobile/WordPress-Android
# Add debug cookies feature – Part 1

This PR creates a `debug` package, moves all the "manual feature configuration" classes into it, and rename them to "debug settings". This is done in preparation for the introduction of the new debug cookies feature as well as other future debug options, besides just manual feature flags.

|<img src="https://user-images.githubusercontent.com/830056/128928736-e0e5b148-8ba0-4729-b42d-4671be6b8f41.png" width=300/>|<img src="https://user-images.githubusercontent.com/830056/128928732-5a706a5c-d415-4aee-acfa-1cf0f4b03985.png" width=300/>|
|----|----|

Most of the changes here are purely moving and renaming classes, so it should be easy to follow, especially through the commit history. As a bonus, I also fixed the method used to restart the app after enabling/disabling a feature as it wasn't working properly before.

To test:

1. Open the app.
1. Tap the Me button.
1. On the Me screen, tap App Settings.
1. On the App Settings screen, notice the "Test feature configuration" item has been renamed to "Debug settings".
1. Tap the Debug Settings item.
1. Notice the "Test feature configuration" screen has been renamed to "Debug settings".
1. Scroll down to the end of the screen.
1. Notice the new Tools section header (it's empty for now).

## Regression Notes
1. Potential unintended areas of impact
Manual feature configuration flags.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Existing unit tests.

3. What automated tests I added (or what prevented me from doing so)
No new tests needed, as this is mostly moving classes around.

PR submission checklist:
